CSS/media.css

```diff
@@ -0,0 +1,283 @@
+@media screen and (max-width: 1116px) {
+  .corner-tnw-img {
+    display: inline-block;
+    width: 114px;
+    height: 80px;
+    background-image: url("../Images/tnw_navbar.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    position: absolute;
+    z-index: 5;
+  }
+
+  .navbar1 {
+    padding-left: 134px;
+  }
+
+  .navbar2 {
+    padding-left: 0;
+  }
+
+  .nv2-img {
+    display: inline-block;
+    width: 114px;
+    height: 48px;
+    background-image: url("../Images/tnw_navbar.png");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .nav2-ul {
+    margin-left: 20px;
+  }
+
+  .nv2-div-tnw,
+  .nv2-div-clock,
+  .nv2-bttn {
+    display: none;
+  }
+
+  .navbar3 {
+    padding-left: 134px;
+  }
+
+  .grid-1 {
+    height: 366px;
+  }
+
+  .grd1-big {
+    background:
+      linear-gradient(170deg, transparent, rgba(0, 0, 0, 0.8)),
+      url(../Images/grid1_big_image.jpg);
+    padding: 14px;
+    position: relative;
+  }
+
+  .big-logo-date {
+    display: none;
+  }
+
+  .big-title {
+    /* position: absolute;
+    bottom: 47px; */
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/